src/Compilers/Core/Portable/MetadataReader/MetadataTypeName.cs

```diff
@@ -5,12 +5,14 @@
 #nullable disable
 
 using System;
+using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis
 {
+
     /// <summary>
```

Comment:
It looks like an empty line is added between `{` and the doc comment. Intentional?

Reply:
done.